xml_inscriptions/transcribed/MI.AA.UM.KM.L.1580.xml

```diff
@@ -128,13 +129,19 @@
         
         
                 <div type="translation">
-                   <p>[   ] and Sextus [   ].</p>
+                   <p>[-] and Sextus [- ---].</p>
                 </div>
             
         
         
                 <div type="commentary">
+
                    <p>Line 2: V Non(a)s S(eptembres) (Tuck).</p>
+                   <p>Line 1: all but the final S of another name is missing.</p>
```

Comment:
Consider rearranging ordering, since it will display in the order which you present the information here.